docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -788,6 +790,40 @@ paths:
       security:
       - {}
       - httpAuth: []
+  /api/migrate/scenario/description/version:
+    get:
+      tags:
+      - Migrations
+      summary: current version of the scenario description version being used
+      operationId: getApiMigrateScenarioDescriptionVersion
+      responses:
+        '200':
+          description: ''
+          content:
+            text/plain:
+              schema:
+                type: integer
```

Comment:
maybe we should move the integer to JSON object? It'd be easier to extend it in a backward compatible way in the future (and costs almost nothing)